test/metabase/analytics/stats_test.clj

```diff
@@ -22,6 +22,13 @@
 
 (use-fixtures :once (fixtures/initialize :db))
 
+(deftest merge-count-maps-test
+  (is (= {:a 1, :b 22, :c 30 :d 1}
+         (#'stats/merge-count-maps [{:a 1, :b 2}
+                                    {:b 20, :c 30}
+                                    {:d "one"}]))
```

Comment:
nit - The `"one" -> 1` conversion here made me wonder if every string value representing an integer is parsed (i.e. `"two" -> 2`, but that's not the case. The implementation just treats any truthy value as 1. Maybe you could change this test case to make that more clear?
```suggestion
                                    {:d true}]))
```

Reply:
ah, sure. I meant this to mean that "one" will be 1, but I see how it could seem like "two" should be 2, as well. I'll remove the ambiguity.